validator/slashing-protection-history/export.go

```diff
@@ -27,7 +28,7 @@ func ExportStandardProtectionJSON(
 	if err != nil {
 		return nil, errors.Wrap(err, "could not get genesis validators root from DB")
 	}
-	if genesisValidatorsRoot == nil || bytes.Equal(genesisValidatorsRoot, params.BeaconConfig().ZeroHash[:]) {
+	if genesisValidatorsRoot == nil || !bytesutil.NonZeroRoot(genesisValidatorsRoot) {
```

Comment:
I think it would be cleaner to have a `bytesutil.ZeroRoot` function instead of using a double negative: `!` and `Non`

Reply:
yeah that's true too should refactor this...

Reply:
fixed